changelog/algorithm_mutation_swapat.dd

```diff
@@ -0,0 +1,4 @@
+Title: `algorithm.mutation.swapAt` was exposed
+Description:
+$(XREF algorithm, mutation, swapAt) allows to swap elements
```

Comment:
I had quite some trouble w/ `REF` in the changelog last time @ntrel.
https://github.com/dlang/phobos/commit/6af6144252bf32746dd96bb5a4d6ff3c0b0e8ea6